hw/Makefile

```diff
@@ -33,6 +33,7 @@ IPS ?= aes           \
        pwrmgr        \
        rom_ctrl      \
        rstmgr        \
+       rv_core_ibex_peri \
```

Comment:
Given that this is a TL "device" that contains CSRs and is not actually a peripheral such as UART / SPI, how about `rv_core_ibex_csrs`? It also makes it clear that we are maintaining CSRs that would otherwise be a part of `rv_core_ibex` in this separate module.

Reply:
it also currently contains the alert senders, so slightly more than just csrs